jena-arq/src/main/java/org/apache/jena/query/ARQ.java

```diff
@@ -460,7 +460,7 @@ public static void enableOptimizer(Context context, boolean state)
     public static final Symbol optMergeBGPs = SystemARQ.allocSymbol("optMergeBGPs");
 
     /**
-     * Context key controlling whether the standard optimizater applies the optimization
```

Comment:
Though a typo, that sounds like a cool name for some project, or sci-fi weapon :rocket: